### ISSUE TITLE
Download only photos after X date?

### ISSUE THREAD

**Issue by bkcberry** [boredazfcuk/docker-icloudpd#363]
Hello, I was wondering if there is an option to only download photos there were taken/created after a certain date?

**Comment by AndreyNikiforov** [comment]
no, filtering by dates is not available. it's been requested multiple times, so we may have this functionality eventually.

**Comment by bkcberry** [comment]
Thanks

**Comment by boredazfcuk** [comment]
Closing this off as there's no point it hanging around. If the functionality is added upstream, I'll incorporate it here.
